### DISCUSSION TITLE
I can't join global campus

### DISCUSSION THREAD

**Discussion by meralhamarat** [community/community#82101] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello! I'm new to GitHub, and I learned that individuals with verified student status can use GitHub Copilot for free. I have verified my student status. When I click on "Join Global Campus," it says "Congrats! Your request for @meralhamarat was approved, and your benefits will be available soon." However, I still don't appear to be approved. Is the free availability of Copilot a lie, and why can't I join the campus? I would be very happy if you could help.

**Comment by faridnec** [comment, marked as answer]
Did they emailed you? in my case they emailed me and they provide a link to [Github Education](https://education.github.com/globalcampus/student). They make a survey and then will shortly train you. On training I got my access to Copilot. maybe check email or github education site! Hope this might help

**Reply by meralhamarat** [reply]
OK, got it. Thanks a lot.

**Comment by qwedc001** [comment]
Sit back and relax.It will take you some time(took me about two days) before all the benefits is being activated.
you can first have a look at https://github.com/settings/copilot (redirected from https://github.com/github-copilot/signup ) ,one of the two links may tell you what to do next.

**Reply by meralhamarat** [reply]
OK, got it. Thanks a lot.
